constructors/brick.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
 const Brick = function(x,y,width,height,id,hp,colors,canvas,ctx){
-  this.x = x
-  this.y = y
+  this.xx = xx
+  this.yy = yy
```

Comment:
Sometimes I like to use "xx" instead of simply "x" because "x" can be hard to search for in a file and "xx" is way more rare and it makes search and replace way easier. Something to consider.